src/directives.ts

```diff
@@ -7,8 +7,11 @@
  * return { reply: 'View' }
  */
 
+import { Response } from "ask-sdk-model";
 import * as bluebird from "bluebird";
 import * as _ from "lodash";
+import { DialogflowReply } from ".";
+import { AlexaReply } from "../src/platforms/alexa/AlexaReply";
```

Comment:
I'm not sure i quite like this, it's platform specific code but we're implementing it in the core part of the framework